src/pages/MonthlyGood/LandingFormExperiment.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+	<form @submit.prevent.stop="submit" novalidate>
+		<div class="row">
+			<multi-amount-selector
+				id="amount-selector"
+				:options="amountOptions"
+				:selected="amountSelected"
+				@pill-toggled="pillToggled"
+				:min-custom-amount="minCustomAmount"
+				:max-custom-amount="maxCustomAmount"
+				@custom-amount-updated="customAmountUpdated"
+				:split-pills="true"
+			/>
+			<span class="large-4 medium-4 small-12 columns select-category-label">
+				Loan monthly to
+			</span>
+			<span class="large-7 medium-7 small-12 columns select-category">
+				<select
+					class="loan-price medium-text-font-size"
+					@change="updateSelected"
+				>
+					<option
+						v-for="(option, index) in lendingCategories"
+						:value="option.value"
+						:key="index"
+					>
+						{{ option.label }}
+					</option>
+				</select>
+			</span>
+		</div>
+
+		<kv-button class="smaller" type="submit" :disabled="$v.$invalid">
+			{{ buttonText }}
+		</kv-button>
+	</form>
+</template>
+
+
+<script>
+import { validationMixin } from 'vuelidate';
+import { required, minValue, maxValue } from 'vuelidate/lib/validators';
+import { number } from '@storybook/addon-knobs';
+import KvButton from '@/components/Kv/KvButton';
+import loanGroupCategoriesMixin from '@/plugins/loan-group-categories';
+import MultiAmountSelector from '@/components/Forms/MultiAmountSelector';
+
+export default {
+	mixins: [
+		validationMixin,
+		loanGroupCategoriesMixin
+	],
+	components: {
+		KvButton,
+		MultiAmountSelector
+	},
+	validations: {
+		amount: {
+			required,
+			minValue: minValue(5),
+			maxValue: maxValue(10000)
+		},
+	},
+	data() {
+		return {
+			amountOptions: [
+				{
+					title: '$5',
+					key: '5',
+				},
+				{
+					title: '$10',
+					key: '10',
+				},
+				{
+					title: '$25',
+					key: '25',
+				},
+				{
+					title: '$50',
+					key: '50',
+				},
+				{
+					title: 'Other',
+					key: 'custom'
+				}
+			],
+			amountSelected: '25',
+			customAmount: 5,
+		};
+	},
+	props: {
+		amount: {
+			type: Number,
+			default: 25
+		},
+		selectedGroup: {
+			type: String,
+			default: 'default'
+		},
+		buttonText: {
+			type: String,
+			default: 'Contribute monthly'
+		},
+		minCustomAmount: {
+			type: Number,
+			default: number('minCustomAmount', 5)
```

Comment:
@mlibazisi you can just put the number 5 here... and 10000 below. No need to wrap it